### PR TITLE
Freeze to django-mptt version to fix set+ tuple error

### PR DESCRIPTION
**Pull request by mustyoshi**
It took me spinning up two fresh machines to narrow this down.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -8,8 +8,8 @@
     include_package_data=True,
     install_requires=[
         'cached_property',
-        'django>=1.8,<1.12',
-        'django-mptt',
+        'django==1.11',
```
Comment (cmc333333) [inline setup.py]: I suspect you didn't intend to pin to exactly 1.11 (which excludes 1.11.1, 1.11.2, etc.). As a best effort, we should also _try_ to support 1.8 through 1.10 until we hit an incompatibility. I think that means we can revert this line?

File: setup.py [hunk 2]
```diff
@@ -8,8 +8,8 @@
...
+        'django-mptt==0.8.7',
```
Comment (cmc333333) [inline setup.py]: Sounds like 0.9 caused the issue, so we'll want to exclude that. For maximum compatibility with downstream projects, though, we'll want to expand this version span. Does `django-mptt~=0.8.3` work? That's should match 0.8.3 (the version available when we added this dependency) through 0.8.7.

**Review by cmc333333 — COMMENTED** [review]
Thanks for finding this, @mustyoshi! I think we'll need to expand the supported requirements a bit, but otherwise we can get this in and released ASAP.

File: setup.py (hunk 2 above)
Comment (mustyoshi) [reply setup.py]: I'll test and make another commit tonight.

Comment (mustyoshi) [reply setup.py]: So on my ubuntu 16.04 machine, running Python 3.6

This is just building the docker image
```docker build . -t eregs/core```

django==1.9 works on mptt>=0.8.3,<=0.8.7
django==1.10 works on mptt>=0.8.6,<=0.8.7
django==1.11 works on mptt>=0.8.6,<=0.8.7

I'm not too familiar with Python setuptools, is there a way to make conditional library versioning?

Comment (cmc333333) [reply setup.py]: Womp womp. Thanks so much for testing all of that! Looks like we'll get the max coverage with
```
django >= 1.10, <1.12
django-mptt ~= 0.8.6
```
I've opened #87 for removing mptt going forward.

**Comment by cmc333333** [comment]
Thanks so much @mustyoshi !

**Comment by johnnyporkchops** [comment]
@mustyoshi , At FEC, we forked the [eregs/regluations-core ](https://github.com/fecgov/regulations-core) repo and tried to update python packages like Django and django-mptt.

- Migrations only work with `django-mptt 0.8.6` on fec-eregs repo. regcore.0003_mptt_copy_children fails with any later version of django-mptt.
-  When we upgrade django-mptt (0.9.0/0.11.0) on regulations-core, migrations work. 
The problem is that on fec-eregs migrations continue to fail with the `set+ tuple error` that you fixed with this PR.

We are stumped! Since you worked on this before,  do you know what the nature of the error is and can you think of any possible workarounds?

**Comment by mustyoshi** [comment]
@johnnyporkchops, unfortunately I am not very familiar with the django/django-mptt ecosystem, I had only been messing around with this repo.
I did some testing, and if you keep the django-mptt version at 0.8.6, the docker image will finish building. I was also able to start the webserver.

cmc was in the process of depreciating the use of django-mptt, so you may have to just live with the outdated version. The good news is that django@2.2.11 does work with django-mptt@0.8.6

Sorry I couldn't be more helpful.

**Pull request merged**